src/node/desktop/src/ui/utils.ts

```diff
@@ -40,3 +41,13 @@ export const checkForNewLanguage = () => {
     }, 100);
   });
 };
+
+// Makes sure that the path only has `/` instead of `\` or `\\` or `//`
+function normalizeSeparators(path: string, separator = '/') {
```

Comment:
Why does this function (and the next one) exist in both utils.ts and file-path.ts?

Reply:
As it's being used in the `UI` folder, sometimes code from other parts of the code are not available at some point, like in some standalone pages. Kinda like another module. Had some issues with it previously, but also realized that for this part, there are some code being used outside the `ui` folder. So I'm gonna test removing that and leaving in only one place.

Reply:
`renderer` folder cannot access some node packages. As `file-path.ts do contain some, it's best to have a different file for functions that are strictly non-node dependent